modules/core/src/v2/coins/xrp.ts

```diff
@@ -485,32 +487,32 @@ export class Xrp extends BaseCoin {
    * @param callback
    */
   public recover(params: RecoveryOptions, callback): Bluebird<RecoveryInfo | string> {
```

Comment:
callback param should be `callback?: NodeCallback<RecoveryInfo | string>`

Reply:
ok, fixed